python/pipeline/vreso.py

```diff
@@ -8,8 +8,10 @@
 from pipeline.utils import galvo_corrections, signal, quality, mask_classification, performance, h5
 
 # external storage place
-dj.config['external'] = {'protocol': 'file',
-                         'location': '/mnt/dj-stor01/pipeline-externals'}
+dj.config['stores'] = {'external': {'protocol': 'file', 'location': '/mnt/dj-stor01/pipeline-externals'}}
+
+# dj.config['external'] = {'protocol': 'file',
+#                          'location': '/mnt/dj-stor01/pipeline-externals'}
 
 schema = dj.schema("pipeline_vreso", locals(), create_tables=False)
```

Comment:
update by remove `locals` and also no need for `create_tables=False` here. Please apply this everywhere else, including places that I'm not commenting on.